Libraries/src/Amazon.Lambda.AspNetCoreServer/Internal/InvokeFeatures.cs

```diff
@@ -194,6 +223,50 @@ internal Task ExecuteAsync()
 
         #endregion
 
+        #region IHttpResponseBodyFeature
+#if !NETCOREAPP_2_1
+        Stream IHttpResponseBodyFeature.Stream => ((IHttpResponseFeature)this).Body;
+
+        private PipeWriter _pipeWriter;
+
+        PipeWriter IHttpResponseBodyFeature.Writer
+        {
+            get
+            {
+                if (_pipeWriter == null)
+                {
+                    _pipeWriter = PipeWriter.Create(((IHttpResponseBodyFeature) this).Stream);
+                }
+                return _pipeWriter;
+            }
+        }
+
+        Task IHttpResponseBodyFeature.CompleteAsync()
+        {
+            return Task.CompletedTask;
+        }
+
+        void IHttpResponseBodyFeature.DisableBuffering()
+        {
+            
+        }
+
+        Task IHttpResponseBodyFeature.SendFileAsync(
+            string path,
+            long offset,
+            long? count,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            throw new NotImplementedException("SendFileAsync is not implemented for Serverless ASP.NET Core functions");
+        }
+
+        Task IHttpResponseBodyFeature.StartAsync(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
OnStarting events are triggered in the `AbstractAspNetCoreFunction.ProcessRequest` method. The starting and complete events are a little weird in this context because they are only triggered before and after the response from ASP.NET Core is marshaled into the Lambda request. Nothing is sent to the client until the function is complete and the Lambda response is returned to either API Gateway or Application Load Balancer.